content/ngf/installation/installing-ngf/deploy-data-plane.md

```diff
@@ -0,0 +1,244 @@
+NAME         TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE
+cafe-nginx   LoadBalancer   10.96.172.204   <pending>     80:32615/TCP   3h5m
+```
+
+### How to set annotations and labels on provisioned resources
+
+While the majority of configuration will happen on the NginxProxy resource, that is not always the case. Uniquely, if
+you want to set any annotations or labels on the NGINX Deployment or Service, you need to set those annotations on the Gateway which
+provisioned them. 
+
+To do so, we can use `kubectl edit` on our gateway and add the following to the `spec`:
+
+```yaml
+infrastructure:
+  annotations:
+    annotationKey: annotationValue
+  labels:
+    labelKey: labelValue
+```
+
+After saving the changes, we can check our NGINX Deployment and Service using `kubectl describe`. 
+
+```text
+~ ❯ kubectl describe deployment cafe                                                                                                                                         1m 52s ⎈ kind-kind
+Name:                   cafe-nginx
+Namespace:              default
+CreationTimestamp:      Mon, 05 May 2025 16:49:33 -0700
+...
+Pod Template:
+  Labels:           app.kubernetes.io/instance=my-release
+                    app.kubernetes.io/managed-by=my-release-nginx
+                    app.kubernetes.io/name=cafe-nginx
+                    gateway.networking.k8s.io/gateway-name=cafe
+                    labelKey=labelValue
+  Annotations:      annotationKey: annotationValue
+                    prometheus.io/port: 9113
+                    prometheus.io/scrape: true
+...
+```
+
+{{< note >}} In order for the changes to propagate onto the Service, it needs to be manually restarted. {{< /note >}}
+
+```text
+~ ❯ kubectl describe service cafe-nginx                                                                                                                                             ⎈ kind-kind
+Name:                     cafe-nginx
+Namespace:                default
+Labels:                   app.kubernetes.io/instance=my-release
+                          app.kubernetes.io/managed-by=my-release-nginx
+                          app.kubernetes.io/name=cafe-nginx
+                          gateway.networking.k8s.io/gateway-name=cafe
+                          labelKey=labelValue
+Annotations:              annotationKey: annotationValue
+```
+
+---
+
+## See Also
+
+For more guides on routing traffic to applications and more information on Data Plane configuration, check out the following resources:
+
+- [Routing Traffic to Your App]({{< ref "/ngf/how-to/traffic-management/routing-traffic-to-your-app.md" >}})
+- [Advanced Routing]({{< ref "/ngf/how-to/traffic-management/advanced-routing.md" >}})
+- [Data Plane Configuration]({{< ref "/ngf/how-to/data-plane-configuration.md" >}})
+- [NGINX Gateway Fabric API Reference]({{< ref "/ngf/reference/api.md" >}})
```

Comment:
```suggestion
- [Routing traffic to applications]({{< ref "/ngf/how-to/traffic-management/routing-traffic-to-your-app.md" >}})
- [Application routes using HTTP matching conditions]({{< ref "/ngf/how-to/traffic-management/advanced-routing.md" >}})
- [Data plane configuration]({{< ref "/ngf/how-to/data-plane-configuration.md" >}})
- [API reference]({{< ref "/ngf/reference/api.md" >}})
```

Links should match their titles, and all headings are in sentence case, not title case.